docs/protocol/Validators.md

```diff
@@ -1,41 +1,48 @@
+- `set`       (VS):     Validator set being joined.
 
-`Stake` specifies a validator, or validator candidate, and delegates the specified amount to them. Initially, only the manager of a validator may stake to them. The stake amount must be a non-zero multiple of 1m $SRI. This is due to multisig key shares being without weights, requiring validators to be equal accordingly to ensure security. Each 1m $SRI causes a new share of the group key to be assigned.
+Marks `validator` as a validator candidate for the specified validator set,
+enabling delegation.
 
-# Resign (message)
+### Delegate (message)
 
-`Resign` specifies a validator, or validator candidate, and initiates their removal. For an active validator, they will be removed at the next churn, returning their bond to the validator's manager. For a non-active validator, their bond is immediately returned.
+  - `delegator` (signer):  Address delegating funds to `validator`.
+  - `validator` (address): Registered validator being delegated to.
+  - `amount`    (Amount):  Amount of funds being delegated to `validator`.
 
-## Rationale for the separation of Register and Stake
+Delegated funds will be removed from `delegator`'s wallet and moved to
+`validator`'s bond. If `validator`'s bond is not a multiple of the validator
+set's bond, it is queued, and will become actively delegated when another
+delegator reduces their bond.
 
-Ideally, a DPoS system will eventually be possible where stake takes a variadic amount. Once the stake value hits 1m, the validator is considered for active inclusion. If the stake value hits a new multiple of 1m, a new key share would be assigned. Any excess stake amount would be added to a queue and when someone else unstakes, this queue will shift in. If there is insufficient excess stake, the node would lose a key share/inclusion in validation entirely. This is not being considered for the initial release, leaving out an `Unstake` message and instead routing handling though `Resign`.
+Note: At launch, only `validator`'s manager will be able to delegate to
+`validator`, and only in multiples of the validator set's bond.
 
-# Oraclization (message)
+### Undelegate (message)
 
-`Oraclization` messages are published by the current block producer and communicate an external event being communicated to the native chain. This is presumably some other cryptocurrency, such as BTC, being sent to the multisig wallet, triggering a privileged call enabling relevant actions.
+  - `delegator` (signer):  Address removing delegated funds from `validator`.
+  - `validator` (address): Registered validator no longer being delegated to.
+  - `amount`    (Amount):  Amount of funds no longer being delegated to
+`validator`.
 
-# Report (message)
+If a sufficient amount of funds are queued, the `validator`'s operation
+continues normally, shifting in queued funds. If the `validator` falls below a
+multiple of the validator set's bond, they will lose a key share at the next
```

Comment:
Seems we could take queued bond into account when distributing rewards and selecting validators to churn in while funds stay protected by unit bond. Thoughts?

Reply:
The entire queue system was removed as part of #55. If/when it is re-introduced, I'd rather not consider it for rewards in order to have people actively work towards making it active, rotating it as needed.